.github/workflows/test-cov.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
       - uses: actions/setup-go@v5
         with:
-          go-version: "1.23.5"
+          go-version: "1.23.6"
```

Comment:
can we use only the latest stable version here?

```suggestion
          go-version: "stable"
```

Reply:
No, see https://github.com/ignite/cli/pull/4509